.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,72 @@
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  php-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.4, 7.3, 7.2]
+        laravel: [7.*]
```

Comment:
the composer.json file claims support for `^5.5 || ^6 || ^7"`, do we want to expand this matrix here too? It looks like it might be an oversight here that we are missing them, as they are in the `includes` section below 😄

Reply:
Maybe it's smarter to actually define all things in the matrix and rather use exclude, would that work?

Reply:
I think it's a bit overkill to test all those combinations, that's why I left them out the matrix, but included those 2 in the includes.

Reply:
Figured that might be the case, need to draw a line somewhere 👍